packages/map-template/src/components/QRCodeDialog/QRCodeDialog.jsx

```diff
@@ -32,11 +32,15 @@ function QRCodeDialog() {
 
     useEffect(() => {
         if (directionsFrom && directionsTo) {
-            // The target URL when the user opens the QR code dialog
-            // Create the URL by combining the origin and the pathname
-            // Check if the pathname is a '/' before appending it.
-            let targetUrl = window.location.origin + (window.location.pathname === '/' ? '' : window.location.pathname)
+            // Get the last character of the pathname
+            const lastCharacter = window.location.pathname.slice(-1);
```

Comment:
I don't think we need this anymore. The regular expression should take care of it.